src/components/icon/Icon.tsx

```diff
@@ -33,18 +33,19 @@ const Icon: React.FC<IconProps> = ({
   onKeyPress,
   forwardRef,
   tabIndex,
-  ...rest
+  style,
+  ...otherProps
 }) => {
   const click = onClick ? onClick : handleClick;
   return (
     <i
-      {...rest}
+      {...otherProps}
       className={classnames(`tk-icon-${iconName}`, className)}
       onClick={!disabled ? click : null}
       onKeyDown={!disabled ? onKeyDown : null}
       onKeyPress={!disabled ? onKeyPress : null}
       ref={forwardRef}
-      style={{ cursor: !disabled && click && 'pointer' }}
+      style={{ cursor: !disabled && click && 'pointer', ...style }}
```

Comment:
Why do we need to pass inline styles? Isn't the classname enough for adding custom stylings?

Reply:
`style` is one of the `React.HTMLProps` and it should work passing inline-styles. className is of course preferred.

Reply:
The reason why it doesn't work is cause we're overriding the style with our own, that's why I have to break it out and pass it in the existing style prop.